selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -176,28 +177,38 @@ def test_fw_requests(self):
 
 class TestFwFingerprintTiming(unittest.TestCase):
   N: int = 5
-  TOL: float = 0.12
+  TOL: float = 0.1
 
   @staticmethod
   def _run_thread(thread: threading.Thread) -> float:
     params = Params()
     params.put_bool("ObdMultiplexingEnabled", True)
     thread.start()
     t = time.perf_counter()
+
     while thread.is_alive():
-      time.sleep(0.02)
+      time.sleep(0.001)
```

Comment:
Can you see if the change in timings for OBD multiplexing matches what it is when fingerprinting on device (think going onroad on a desk is enough)?